### ISSUE TITLE
Typos in citation

### ISSUE THREAD

**Issue by KevinShook** [brentthorne/posterdown#70]
Pretty minor - should be "Where" instead of "Were", and "Won't" instead of "Wont".
Great stuff BTW - hoping to use the templates very soon.
Thanks for all this work. 

W. Brent Thorne (2019). posterdown: An R Package Built to Generate Reproducible Conference Posters for the Academic and Professional World Were Powerpoint and Pages Just Wont Cut It. R package version 0.1.2.

A BibTeX entry for LaTeX users is:

  @Manual{posterdown,
    title = {posterdown: An R Package Built to Generate Reproducible Conference Posters for the Academic and Professional World Were Powerpoint and Pages Just Wont Cut It},
    author = {W. Brent Thorne},
    year = {2019},
    url = {https://github.com/brentthorne/posterdown},
    note = {R package version 0.1.2},
  }

**Comment by brentthorne** [comment]
Thanks for this, I will always be greatful for those who are able to find my many typos!

**Comment by KevinShook** [comment]
Quick work! 
I see you followed my page.
As you can see, I'm part of the team putting together the CSHS_hydRology package.
We're working on a package for Canadian hydrologists. It's early days so far,
but we are hoping to expand into Ecohydrology. Dan Moore from UBC is heading the 
spatial theme, which might interest you. The plan is to submit the package to CRAN.

We're always looking for new people to get involved. Right now we have a monthly conference
call to keep in touch, and we are also running some sessions at the CWRA conference in
Collingwood.
If you are interested at all, you can give me an email at kevin.shook@usask.ca.
BTW, do you accept 1 time payments through PayPal?
I'm starting some R training next month and if it's OK with you I'd like to
show my students your poster package as a part of teaching R Markdown.
K

**Comment by brentthorne** [comment]
I knew I had to give you a follow when I saw all the shell and bash scripts you have! I will certainly send an email to you about the CSHC_hydRology package and possible availability to help out where I can.

I haven't used it yet but I did set up a paypal account and it has a link of [paypal.me/brentthorne](https://www.paypal.me/brentthorne) for making one time payments. Let me know if there are any issues with this as you are the first person to offer any kind of donation, so a huge thank you for that! 

Feel free to ask me here or by email if you have any questions and or requests for {posterdown} leading up to your R training. Also, if your students are writing a thesis and are interested in {rmarkdown} then have also made a package match specific style guides of a my current university: [{brockdown}](https://github.com/brentthorne/brockdown), which can be changed to match U of S formatting quite simply.

Thanks again for your interest and support!

Brent
